packages/belos/tpetra/src/solvers/Belos_Tpetra_GmresSstep.hpp

```diff
@@ -240,16 +1012,78 @@ class GmresSstep : public Gmres<SC, MV, OP>  {
       = params.get<bool> ("Compute Ritz Values on Fly", this->input_.computeRitzValuesOnFly);
     this->input_.computeRitzValuesOnFly = computeRitzValuesOnFly;
 
+    // load intra block ortho option (CholQR is default)
+    std::string orthoType = this->input_.orthoType;
+    constexpr bool useCholQR2_default = false;
+    bool useCholQR2 = params.get<bool> ("CholeskyQR2", useCholQR2_default);
```

Comment:
Why two versions here?

Reply:
I am not sure what "versions" but CholQR2 runs CholQR twice.